<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="SampleTest"
           tests="2"
           failures="1"
           errors="0"
           skipped="1"
           time="0.000">
    <properties></properties>

            <testcase
                name="AdminTest_01"
                time="0.000"
                classname="Test.AdminWebTest">

            <failure type="org.testng.TestNGException" message="
Method public void Test.AdminWebTest.AdminTest_01(java.lang.String,java.util.HashMap) throws java.net.MalformedURLException,java.lang.InterruptedException requires a @DataProvider named : Test_data">
                <![CDATA[org.testng.TestNGException: 
Method public void Test.AdminWebTest.AdminTest_01(java.lang.String,java.util.HashMap) throws java.net.MalformedURLException,java.lang.InterruptedException requires a @DataProvider named : Test_data
                org.testng.internal.Parameters.findDataProvider(Parameters.java:526)org.testng.internal.Parameters.handleParameters(Parameters.java:763)org.testng.internal.Parameters.handleParameters(Parameters.java:728)org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:911)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)java.base/java.util.ArrayList.forEach(ArrayList.java:1511)org.testng.TestRunner.privateRun(TestRunner.java:848)org.testng.TestRunner.run(TestRunner.java:621)org.testng.SuiteRunner.runTest(SuiteRunner.java:443)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)org.testng.SuiteRunner.run(SuiteRunner.java:336)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)org.testng.TestNG.runSuitesLocally(TestNG.java:1200)org.testng.TestNG.runSuites(TestNG.java:1114)org.testng.TestNG.run(TestNG.java:1082)com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
            </failure>
    </testcase>

            <testcase
                name="Setup"
                time="0.0"
                classname="Test.AdminWebTest">
            </testcase>

</testsuite>
